.github/workflows/image-build.yml

```diff
@@ -1,12 +1,7 @@
-name: 'Build and publish image'
+name: "Build and publish image"
```

Comment:
Correct me if I am wrong but this action is no longer needed since we will be using central repository and therefore we don't have to push to multiple repositories